README.md

```diff
@@ -129,59 +147,57 @@ This creates a journal entry between two accounts.
 ```js
 const transfer = {
     id: 1n, // u128
+    // T-based accounting:
     debit_account_id: 1n,  // u128
     credit_account_id: 2n, // u128
-    user_data: 0n, // u128, opaque third-party identifier to link this transfer (many-to-one) to an external entity 
-    reserved: Buffer.alloc(32, 0), // two-phase condition can go in here
-    timeout: 0n, // u64, in nano-seconds. 
-    code: 1,  // u32, a chart of accounts code describing the reason for the transfer (e.g. deposit, settlement)
+    // Opaque third-party identifier to link this transfer to an external entity:
+    user_data: 0n, // u128, (many-to-one)  
```

Comment:
I'm not sure I understand what this "many-to-one" comment is trying to say. Is it not entirely up to the user how the `user_data` field is used?

Reply:
You are correct. I removed it in the two places.